drivers/storage/portworx/portworx.go

```diff
@@ -33,8 +33,10 @@ const (
 	defaultSDKPort                    = 9020
 	defaultSecretsProvider            = "k8s"
 	defaultNodeWiperImage             = "portworx/px-node-wiper:2.1.4"
+	pxAuthSecretKey                   = "auth-secret"
 	envKeyNodeWiperImage              = "PX_NODE_WIPER_IMAGE"
 	envKeyPortworxEnableTLS           = "PX_ENABLE_TLS"
+	envKeyPortworxSharedSecretKey     = "PORTWORX_AUTH_SYSTEM_KEY"
```

Comment:
This is the same env variable that is passed to PX, right? Just want to make sure we are not adding any special env variables.

Reply:
Yes, confirmed this by testing if auth was enabled after each setup - direct env value, secretRef, and configMapRef. PX started up with auth enabled and the operator authenticated with the SDK too.